index.bs

```diff
@@ -1099,20 +1103,28 @@ optional |expires|,
 |sameSite|,
 run the following steps:
 
+1. If |name|, |value|, |domain|, or |path| contain U+003B (`;`), any [=C0 control=] character, or U+007F, then return failure.
 1. If |name|'s [=string/length=] is 0 and |value| contains U+003D (`=`), then return failure.
 1. If |name|'s [=string/length=] is 0 and |value|'s [=string/length=] is 0, then return failure.
+1. Let |encodedName| be the result of [=UTF-8 encode|UTF-8 encoding=] |name|.
+1. Let |encodedValue| be the result of [=UTF-8 encode|UTF-8 encoding=] |value|.
+1. If the [=byte sequence=] [=byte sequence/length=] of |encodedName| plus the [=byte sequence=] [=byte sequence/length=] of |encodedValue| is greater than the <a for=cookie>maximum name/value pair size</a>, then return failure.
```

Comment:
Can you use the `[=cookie/maximum name/value pair size=]` syntax here for consistency?

Reply:
I tried that initially but it looks like the slash breaks things:
```
      LINE ~1111: No 'dfn' refs found for 'value pair size'.
      WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
      <dfn data-dfn-for="cookie" data-dfn-type="dfn" id="cookie-maximum-name-value-pair-size" data-lt="maximum name/value pair size" data-noexport="by-default">maximum name/value pair size<a href="#cookie-maximum-name-value-pair-size" class="self-link"></a></dfn>
```
should we use something else instead of `maximum name/value pair size`?

Reply:
Oh, oops! Stick with the <a> then.